apps/librepcb/controlpanel/controlpanel.cpp

```diff
@@ -122,13 +122,12 @@ ControlPanel::ControlPanel(Workspace& workspace) :
 
     loadSettings();
 
-    // parse command line arguments and open all project files
-    foreach (const QString& arg, qApp->arguments())
-    {
-        FilePath filepath(arg);
-        if ((filepath.isExistingFile()) && (filepath.getSuffix() == "lpp"))
-            openProject(filepath);
-    }
+    // slightly delay opening projects to not block processing of other important events
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 4, 0))
+    QTimer::singleShot(10, this, &ControlPanel::openProjectsPassedByCommandLine);
+#else
+    QTimer::singleShot(10, this, SLOT(openProjectsPassedByCommandLine()));
+#endif
```

Comment:
Do we really want to introduce technical dept like that before the first release? Wouldn't it be easier to just require Qt >= 5.4?
People working on ancient, soon to be unsupported distributions can easily install a newer Qt version from other sources.

Reply:
Discussing this topic (once again...) requires more time than I needed for the implementation ;) I stick with Qt5.2 support until Ubuntu 14.04 support has ended. It's *really* not a big deal to keep Qt5.2 support!

Reply:
So I'll keep mentioning it until April 2019 :wink: . And then I'll start complaining about unnecessary backwards compatibility with Qt < 5.9 :stuck_out_tongue_closed_eyes: